ui/src/App.css

```diff
@@ -0,0 +1,8 @@
+.App-header {
```

Comment:
Rule:
- The standard naming convention for the css classes will contain lowercase letters

Reply:
also the file name should contain lower case letters